plf-tomcat-resources/src/main/resources/bin/setenv.sh

```diff
@@ -133,7 +134,15 @@ fi
 CATALINA_OPTS="$CATALINA_OPTS -Dexo.jcr.session.tracking.active=${EXO_JCR_SESSION_TRACKING}"
 
 # JVM memory allocation pool parameters
-CATALINA_OPTS="$CATALINA_OPTS -Xms${EXO_JVM_SIZE_MIN} -Xmx${EXO_JVM_SIZE_MAX} -XX:MaxPermSize=${EXO_JVM_PERMSIZE_MAX}"
+CATALINA_OPTS="$CATALINA_OPTS -Xms${EXO_JVM_SIZE_MIN} -Xmx${EXO_JVM_SIZE_MAX}"
+
+# PLF-6510: Configure the JVM according to the version
+cmd=$(java -jar $CATALINA_HOME/bin/exo-tools.jar isJava8OrSuperior)
+if [ $? == 0 ]; then
+  CATALINA_OPTS="$CATALINA_OPTS -XX:MaxMetaspaceSize=${EXO_JVM_METASPACE_SIZE_MAX}"
```

Comment:
With the metaspace, now native memory outside of the heap is used for this area, and it's configured to be unbounded by default, so if a classloader is leaky we might not only go OOM with the VM, but with the whole operating system/container.

> MaxMetaspaceSize : This flag provides you flexibility to set the maximum size that you want to allocate for metaspace. You might want to do this to control the space used by your applications (multiple) running on a server or in a case of memory leak so that it won’t use all of available space in native memory. Now in the case when your native memory is fully and your application still needs more space in that case you will see a new error message : java.lang.OutOfMemoryError: Metadata space.